spec.html

```diff
@@ -41705,158 +41858,24 @@ <h1>Additional Syntax</h1>
-          `0`
-          NonZeroDigit
-          NonZeroDigit NumericLiteralSeparator? DecimalDigits[+Sep]
-          NonOctalDecimalIntegerLiteral
-
-        NonOctalDecimalIntegerLiteral ::
-          `0` NonOctalDigit
-          LegacyOctalLikeDecimalIntegerLiteral NonOctalDigit
-          NonOctalDecimalIntegerLiteral DecimalDigit
-
-        LegacyOctalLikeDecimalIntegerLiteral ::
-          `0` OctalDigit
-          LegacyOctalLikeDecimalIntegerLiteral OctalDigit
+      <p>The following syntax from <emu-xref href="#sec-literals-numeric-literals"></emu-xref>, and its associated semantics, used to be normative optional:</p>
+      <emu-grammar>
+        NumericLiteral :: LegacyOctalIntegerLiteral
 
-        NonOctalDigit :: one of
-          `8` `9`
+        DecimalIntegerLiteral :: NonOctalDecimalIntegerLiteral
       </emu-grammar>
-
-      <emu-annex id="sec-additional-syntax-numeric-literals-status-semantics">
-      <h1>Static Semantics</h1>
-      <ul>
-        <li>
-          The MV of <emu-grammar>LegacyOctalIntegerLiteral :: `0` OctalDigit</emu-grammar> is the MV of |OctalDigit|.
-        </li>
-        <li>
-          The MV of <emu-grammar>LegacyOctalIntegerLiteral :: LegacyOctalIntegerLiteral OctalDigit</emu-grammar> is (the MV of |LegacyOctalIntegerLiteral| times 8) plus the MV of |OctalDigit|.
-        </li>
-        <li>
-          The MV of <emu-grammar>DecimalIntegerLiteral :: NonOctalDecimalIntegerLiteral</emu-grammar> is the MV of |NonOctalDecimalIntegerLiteral|.
-        </li>
-        <li>
-          The MV of <emu-grammar>NonOctalDecimalIntegerLiteral :: `0` NonOctalDigit</emu-grammar> is the MV of |NonOctalDigit|.
-        </li>
-        <li>
-          The MV of <emu-grammar>NonOctalDecimalIntegerLiteral :: LegacyOctalLikeDecimalIntegerLiteral NonOctalDigit</emu-grammar> is (the MV of |LegacyOctalLikeDecimalIntegerLiteral| times 10) plus the MV of |NonOctalDigit|.
-        </li>
-        <li>
-          The MV of <emu-grammar>NonOctalDecimalIntegerLiteral :: NonOctalDecimalIntegerLiteral DecimalDigit</emu-grammar> is (the MV of |NonOctalDecimalIntegerLiteral| times 10) plus the MV of |DecimalDigit|.
-        </li>
-        <li>
-          The MV of <emu-grammar>LegacyOctalLikeDecimalIntegerLiteral :: `0` OctalDigit</emu-grammar> is the MV of |OctalDigit|.
-        </li>
-        <li>
-          The MV of <emu-grammar>LegacyOctalLikeDecimalIntegerLiteral :: LegacyOctalLikeDecimalIntegerLiteral OctalDigit</emu-grammar> is (the MV of |LegacyOctalLikeDecimalIntegerLiteral| times 10) plus the MV of |OctalDigit|.
-        </li>
-        <li>
-          The MV of <emu-grammar>NonOctalDigit :: `8`</emu-grammar> is 8.
-        </li>
-        <li>
-          The MV of <emu-grammar>NonOctalDigit :: `9`</emu-grammar> is 9.
-        </li>
-      </ul>
-      </emu-annex>
+      <p>and the productions for |LegacyOctalIntegerLiteral|, |NonOctalDecimalIntegerLiteral|, |LegacyOctalLikeDecimalIntegerLiteral|, and |NonOctalDigit|.</p>
```

Comment:
I think it might read better if we just list all of the alternatives for these productions, in the same way we list the alternatives for the productions above that were only partially optional.

Reply:
Okay. So you're saying that we should keep these tombstones, or you're only saying *if* we keep them, then list the full productions?

Reply:
If we keep them, list the full productions. I have no opinion on keeping them.

Reply:
Preference from editor call was to omit the tombstones.